src/main/java/seedu/address/model/contact/ContactName.java

```diff
@@ -7,7 +7,7 @@
  * Represents a Contact's name in the contact list.
  * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
  */
-public class Name {
+public class ContactName {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Names should only contain alphanumeric characters and spaces, and it should not be blank";
```

Comment:
Perhaps the "Name" mentioned in the message constraints could be changed to Contact Name again. This might help increase readability by further differentiating from Module Name present in the Module List component.

Reply:
Yes, I think it should be changed.